src/com/palmergames/bukkit/towny/object/TownBlockTypeHandler.java

```diff
@@ -51,16 +54,27 @@ public static void initialize() {
 	/**
 	 * Registers a new type. Should not be used at all outside of the TownBlockTypeRegisterEvent.
 	 * @param type - The type
-	 * @throws TownyException - If a type with this name is already registered.
 	 */
 	public static void registerType(@NotNull TownBlockType type) throws TownyException {
-		if (exists(type.getName()))
-			throw new TownyException(String.format("API: A type named '%s' is already registered!", type.getName()));
-		
-		townBlockTypeMap.put(type.getName().toLowerCase(), type);
+		if (isLoadedAlready(type))
```

Comment:
This might cause issues when a plugin overrides one of the base townblocktypes instead of setting the data, in a lot of places we use == instead of .equals for checking the type of a townblock which would start returning false for types changed via /plot set type

Reply:
I had never thought a plugin would override our base townblocktypes, I dont know if we should allow for that.

Reply:
Also part of the reason I went with a new private isLoadedAlready instead of reusing the exists also.